src/Products/Common/Util/Comparator/FileDateComparator.cs

```diff
@@ -16,16 +17,16 @@ public class FileDateComparator : IComparer<string>
         /// <returns></returns>
         public int Compare(string x, string y)
         {
-            string strExt1 = File.GetCreationTime(x).ToString();
-            string strExt2 = File.GetCreationTime(y).ToString();
+            string strExt1 = File.GetCreationTime(x).ToString(CultureInfo.InvariantCulture);
+            string strExt2 = File.GetCreationTime(y).ToString(CultureInfo.InvariantCulture);
 
             if (strExt1.Equals(strExt2))
             {
-                return x.CompareTo(y);
+                return x.CompareTo(y.ToLower(CultureInfo.InvariantCulture));
```

Comment:
Codacy found an issue: [Use 'CompareOrdinal' or 'Compare' with the locale specified instead of 'CompareTo'.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=6332042)